### PR TITLE
A few fixes and improvements

### LINKED ISSUE #101: Colorize's regexes are too restrictive
Something like this doesn't get properly colorized:

```javascript
prog.option ( '--config <path|object>', '' );
```

Because `|` is not included in the allowed characters matched by the regex, for no apparent reason.

### PR DESCRIPTION
**Pull request by fabiospampinato**
Fixes #101 #102 #103.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,11 +9,11 @@
     "test": "./node_modules/mocha/bin/mocha --require ./tests/utils/bootstrap.js --full-trace --recursive -R mocha-unfunk-reporter tests/",
     "test-watch": "npm run lint && ./node_modules/mocha/bin/mocha --require ./tests/utils/bootstrap.js --full-trace --watch --recursive -R mocha-unfunk-reporter tests/",
     "coverage": "./node_modules/.bin/istanbul cover --include-all-sources true -x **/examples/**/* --print both ./node_modules/.bin/_mocha -- --require ./tests/utils/bootstrap.js -R spec --recursive tests/",
-    "commit": "git-cz",
-    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -w -r 0",
-    "precommit": "npm run lint && npm test",
-    "prepush": "npm run lint && npm test",
-    "commitmsg": "validate-commit-msg",
+    "commit": "npm --version",
```
Comment (mattallty) [inline package.json]: Seems like an error to change all this scripts

Comment (fabiospampinato) [reply package.json]: I must have forgotten to put them back.

I've removed them because making commits with Husky was becoming a nightmare.

**Comment by mattallty** [comment]
Also could you fix conflicts in lib/colorful.js

File: package.json (hunk 1 above)
Comment (mattallty) [reply package.json]: well if you commit often I understand, but at the end I find it useful for running test automatically & checking my commit messages

**Comment by fabiospampinato** [comment]
Done.

File: lib/colorful.js [hunk 2]
```diff
@@ -3,13 +3,13 @@
 const chalk = require('chalk');
 
 exports.colorize = (text) => {
-  return text.replace(/<([a-z0-9-_.]+)>/gi, (match) => {
+  return text.replace(/<([^>]+)>/gi, (match) => {
     return chalk.blue(match);
   }).replace(/<command>/gi, (match) => {
     return chalk.magenta(match);
-  }).replace(/\[([a-z0-9-_.]+)\]/gi, (match) => {
+  }).replace(/\[([^\[\]]+)\]/gi, (match) => {
```
[the remark this answers is not in the document]
Comment (mattallty) [reply lib/colorful.js]: Could you please also fix this ?

[the remark this answers is not in the document]
Comment (fabiospampinato) [reply lib/colorful.js]: Done.

**Comment by mattallty** [comment]
One last thing, since https://github.com/mattallty/Caporal.js/pull/117 - Caporal uses `colorette` instead of `chalk`

**Comment by fabiospampinato** [comment]
Switched to `colorette`. I'm not so sure switching to `colorette` was a good idea in the first place though, it's kind of a new project compared to `chalk`, which has been tested for years.

**Comment by mattallty** [comment]
> Switched to `colorette`. I'm not so sure switching to `colorette` was a good idea in the first place though, it's kind of a new project compared to `chalk`, which has been tested for years.

Not wrong, will take a deeper look at both later

**Comment by jorgebucaran** [comment]
@fabiospampinato I am the maintainer of Colorette. If you run into any problems, just ping me.

**Pull request merged**